### PR TITLE
Feature scrum

### PR DESCRIPTION
**Pull request by valotg**
Add scrum description to Task Estimation section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,77 @@ When talking about project or task estimation we obviously think about how the e
 
 [Read more about project and task estimation](https://medium.com/globalluxsoft/time-estimation-in-software-development-a4a495c8eb6c)
 
+### SCRUM method
+
+**What's SCRUM method ?**
+
+Scrum is an agile method dedicated to "project management". This management method, or rather this Project Management Framework, aims to improve the productivity of your team.
+This method includes several roles that the whole team must play and it's very helpfull to the task estimation.
```
Comment (4kh) [inline README.md]: Here, "helpfull" should be replaced by "helpful"

File: README.md [hunk 2]
```diff
@@ -40,6 +40,77 @@ When talking about project or task estimation we obviously think about how the e
...
+- [Demo] A relatively simple test (e.g. export an object in XML then delete it from the database, import it from XML, at the end the object must be in the database). This test is a validation test.
+- [Notes] any other information: clarifications, documentary references
+
+### Sprint planning meeting
+
+Before each sprint, we organize a planning meeting, the sprint planning meeting. This planning meeting selects from the product backlog the most priority requirements for the customer. They will be developed, tested and delivered to the customer at the end of the sprint. They constitute the sprint backlog, a sub-set of the product backlog.
+
+### The scrimmage
+
+During the sprint, a progress meeting (about 15 min) is held every day with all team members to ensure that the sprint objectives will be met, this is the Scrum or Scrum. Each day, after the Scrum meeting, the Scrum Master maintains a chart called the sprint burndown chart. This chart gives a very good view of what has been done and the pace of the team's work. It also makes it possible to anticipate whether all the stories in the Sprint Backlog will be completed at the end of the iteration or not.
```
Comment (4kh) [inline README.md]: Maybe there is something missing there ? "this is the Scrum or Scrum"

**Review by 4kh — COMMENTED** [review]
Good content, there are minor errors to correct and maybe try to add kind of introduction on each sub parts to better link with it ? For example something to link Sprint to Product backlog

File: README.md [hunk 3]
```diff
@@ -40,6 +40,84 @@ When talking about project or task estimation we obviously think about how the e
...
+- Am I having difficulty continuing my work.
+- By doing this exercise daily each team member is aware of what his or her colleagues are doing and can coordinate their work and help or be helped in case of difficulties.
+
+The Scrum Meeting is not a meeting in which problems are solved, but only identified and expressed. The role of the Scrum Master is to provide solutions or to delegate to another team member the resolution of problems raised during the Scrum Meeting. Following this meeting, the Scrum Master updates the burndown chart.
+At the end of a sprint, the client is given a demonstration of the latest developments, the Sprint Review Meeting. It is also an opportunity to make a review of the team's performance and to find points for improvement.
+
+Because of its values, Scrum advocates adaptability, under the effect of the experience acquired and the specificities of the project, which brings it closer to the Toyota production method. Visibility, to evaluate the results of the process. Inspection, to verify deviations from the initial objective.
+
+[Read more about SCRUM Method](https://en.wikipedia.org/wiki/Scrum_(software_development))
+
```
Comment (EtiennePASSOT) [inline README.md:120]: Very good !

**Review by EtiennePASSOT — APPROVED** [review]

**Review by 4kh — APPROVED** [review]

**Pull request merged**